src/o365/spo/commands/feature/feature.ts

```diff
@@ -0,0 +1,4 @@
+export interface Feature {
```

Comment:
Since this file is exporting an interface, the file name should follow the interface casing, being named `Feature.ts`